packages/lism-css/src/scss/_auto_output.scss

```
@@ -5,6 +5,8 @@
 
 @use './query' as query;
 @use './setting' as setting;
+// $default_important は $layer_mode を加味した解決済みの値を mixin から読む（no_layer は常に 1）
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)